lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -472,12 +474,16 @@ interface ReceiveAmount {
 dictionary PrepareReceiveRequest {
     PaymentMethod payment_method;
     ReceiveAmount? amount = null;
+    string? offer = null;
```

Comment:
These two params are only for fetching the invoice from the offer and should be only used from the swapper and not concerning other users (it means we will have to document that these shouldn't be used).
Still not sure about it but perhaps we can add a dedicated method for creating invoice from offer? for the sake of making the user API cleaner?

Reply:
Right now I also lean towards having a separate method.

Reply:
Ok, lean consensus wins. Lets add a method to create the invoice. We only have to document it in the nofication plugin section also

Reply:
Done in [82d1386](https://github.com/breez/breez-sdk-liquid/pull/882/commits/82d1386b65164b10910d6f5d73fdcbac39e93262)